recipes-core/os-release/os-release.bbappend

```diff
@@ -20,6 +20,8 @@ OS_RELEASE_FIELDS += "${@bb.utils.contains('OMNECT_RELEASE_IMAGE', '1', '', 'OMN
 
 OS_RELEASE_FIELDS += "OMNECT_RELEASE_IMAGE"
 
+OS_RELEASE_FIELDS += "TUNE_ARCH"
```

Comment:
should we use TARGET_ARCH instead TUNE_ARCH or both yocto variables to os-release?
https://docs.yoctoproject.org/ref-manual/variables.html?highlight=tune_arch#term-TARGET_ARCH

Reply:
done